app/views/users/show.html.erb

```diff
@@ -0,0 +1,3 @@
+<!-- Placeholder for user details -->
+<h1>User Details</h1>
+<p>This is where user details will be displayed.</p>
```

Comment:
- [ ] Kindly add a view `index.html.erb` corresponding to the `index` action created in the user controller